### ISSUE TITLE
exception AVCaptureSession addInput

### ISSUE THREAD

**Issue by dimohamdy** [imaginary-cloud/CameraManager#68]
i get this exception: 
**\* -[AVCaptureSession addInput:] Multiple audio/video AVCaptureInputs are not currently supported
 from this code

```
                if let validBackDevice = _deviceInputFromDevice(backCameraDevice) {
                    if !inputs.contains(validBackDevice) {
                        validCaptureSession.addInput(validBackDevice)
                    }
                }
```

**Comment by torrao** [comment]
@dimohamdy 

this should be fixed in v3.1.4. Thanks for your input 👍

**Comment by vitthalAnpat** [comment]
Hi 
I am using pod 'CameraManager', '~> 4.4' this version but I am facing Multiple audio/video AVCaptureInputs are not currently supported issue 
Can you help me. 
I have attached screenshot for you reference.

![Screenshot 2019-09-09 at 5 00 54 PM](https://user-images.githubusercontent.com/51405209/64527407-7878a780-d323-11e9-8023-5363cb228f53.png)
